generator/dryrun.py

```diff
@@ -38,7 +38,8 @@ class DryRunError(Exception):
 
     def __init__(self, message, error, use_cloud_function, table_id):
         """Initialize DryRunError."""
-        super().__init__(message)
+        # super().__init__(message, error, use_cloud_function, table_id)
+        super(DryRunError, self).__init__(message, error, use_cloud_function, table_id)
```

Comment:
All arguments need to be passed, otherwise due to the use of multiprocessing a pickling error will be thrown

Reply:
Added a comment and changed it to override `__reduce__`